File: window/javascript/gtk/window/window.js
imports.gi.versions.Gtk = "3.0";
const Gtk = imports.gi.Gtk;
Gtk.init(null);

let win = new Gtk.Window();
win.set_title("Empty Window");
win.set_default_size(320, 200);
win.connect("delete-event", () => Gtk.main_quit());
win.show_all();
Gtk.main();

File: window/python/gtk/window/window.py
#! /usr/bin/env python3

import gi
gi.require_version('Gtk', '3.0')
from gi.repository import Gtk

window = Gtk.Window()
window.set_title("Empty Window")
window.set_default_size(320, 200)
window.connect("delete-event", Gtk.main_quit)
window.show()
Gtk.main()
